ext/stringio/extconf.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: false
 require 'mkmf'
 if RUBY_ENGINE == 'ruby'
+  have_func("rb_str_chilled_p", "ruby.h")
```

Comment:
Any reason to specify `ruby.h` here? I think it's unusual to pass it explicitly and not sure of the effect of it.